src/widget/wlibrarysidebar.cpp

```diff
@@ -239,12 +239,36 @@ void WLibrarySidebar::keyPressEvent(QKeyEvent* event) {
     // TODO(XXX) Should first keyEvent ensure previous item has focus? I.e. if the selected
     // item is not focused, require second press to perform the desired action.
 
-    // make the selected item the navigation starting point
+    SidebarModel* sidebarModel = qobject_cast<SidebarModel*>(model());
+    QModelIndexList selectedIndices = selectionModel()->selectedRows();
+    if (sidebarModel && !selectedIndices.isEmpty()) {
+        QModelIndex index = selectedIndices.at(0);
+        if (event->matches(QKeySequence::Delete) || event->key() == Qt::Key_Backspace) {
+            sidebarModel->clear(index);
+            return;
+        }
+        if (event->matches(QKeySequence::Cut)) {
+            sidebarModel->cut(index);
```

Comment:
It feels wrong that the clipboard is not cleared, when trying to copy the whole playlist.

Reply:
I don;t think this does anything

Reply:
My original expectation was that we copy the whole playlist when Crl+C a playlist in the sidebar. 
But that would be a new feature. For now the behaviour is OK after retesting and we can resole it.